uxarray/core/dataarray.py

```diff
@@ -578,6 +582,42 @@ def difference(self, destination: Optional[str] = "edge"):
 
         pass
 
+    def zonal_mean(self,step_size=1):
+        """Computes the Zonal Mean ...
+
+        Can look at the above gradient, difference, or nodal_average
+        function as a reference.
+        """
+
+        # Call zonal average on the data stored under a UxDataArray
+        # data is accessed with self.data or self.values
+        # TODO: Utilize the Grid.bounds method to obtain the latitude bounds of the grid cells
+        data = self.values
+        face_bounds = self.uxgrid.bounds.values
+        faces_lonlat = self.uxgrid.face_lonlat.values
```

Comment:
This should be something like the following (depends on what shape you want `faces_lonlat` to be
```Python
face_lon = self.uxgrid.node_lon.values
face_lat = self.uxgrid.node_lat.values
face_lonlat = np.vstack([face_lon, face_lat]).T
```